client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java

```diff
@@ -53,6 +48,10 @@ public void processOpts() {
         this.testFolder = "";
         this.embeddedTemplateDir = "templates";
 
+        Boolean beanValidation = (Boolean) this.additionalProperties.get("use-bean-validation");
+        this.setUseBeanValidation(beanValidation != null && beanValidation);
+        this.setPerformBeanValidation(beanValidation != null && beanValidation);
```

Comment:
You can use `getOrDefault` and then you won't need to check nullability.

```suggestion
        Boolean beanValidation = (Boolean) this.additionalProperties.getOrDefault("use-bean-validation", false);
        this.setUseBeanValidation(beanValidation);
        this.setPerformBeanValidation(beanValidation);
```